package.json

```diff
@@ -9,11 +9,11 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 5.4.1",
+    "node": "10",
     "yarn": ">= 1.17.3"
   },
   "scripts": {
-    "postinstall": "node --preserve-symlinks --preserve-symlinks-main tools/bazel/postinstall-patches.js",
+    "postinstall": "node --preserve-symlinks --preserve-symlinks-main tools/bazel/postinstall-patches.js | ivy-ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
```

Comment:
I think the issue for the bullet point where the `NgModule`s cannot be loaded in the browser is that we rely in our dev-app on UMD bundles. This is not a bad thing as we do not have tree-shaking anyway for our dev-app.

So, I think the solution would be to process UMD here is as well. I guess the `--first-only` flag would need to be removed. I'm also thinking what stuff we need. Since we only use the Angular framework packages, we should only need `umd` format and the typings transformed. Also I don't think we benefit from the `--create-ivy-entry-points` flag (it rather slows us down)

Reply:
And `--create-ivy-entry-points` does more than slow us down, it's actually the cause of the page not loading and throwing an exception. It appears that removing this and having it create the Ivy entry points in place, rather than creating new entry points and adding them to `package.json`, is what we need for our SystemJS implementation (dev-app).

Reply:
Yeah, we just rely on the original UMD bundles and not look for them in the new ivy ngcc folder.